Command.md

```diff
@@ -0,0 +1,309 @@
+<!-- omit in toc -->
+# Developer guide
+
+This guide helps you get started developing kubi
+
+Make sure you have the following dependencies installed before setting up your developer environment:
+
+ - Git
+ - Jq
+ - Go
```

Comment:
openssl, curl, and wget seems to be required, but are not listed. 

Don't we need docker to build an image too?